### ISSUE TITLE
`lookup_users()` parsing fails on users with no tweets

### ISSUE THREAD

**Issue by alexpghayes** [ropensci/rtweet#597]
``` r
library(rtweet)

packageVersion("rtweet")
#> [1] '0.7.0.9000'

lookup_users("ConvertedBook", parse = FALSE)
#> [[1]]
#>             id              id_str       name   screen_name location
#> 1 1.407792e+18 1407791753470496769 Neil Hoyne ConvertedBook         
#>   description url urls protected followers_count friends_count listed_count
#> 1              NA NULL     FALSE               2             0            0
#>                       created_at favourites_count utc_offset time_zone
#> 1 Wed Jun 23 20:05:16 +0000 2021                0         NA        NA
#>   geo_enabled verified statuses_count lang contributors_enabled is_translator
#> 1       FALSE    FALSE              0   NA                FALSE         FALSE
#>   is_translation_enabled profile_background_color profile_background_image_url
#> 1                  FALSE                   F5F8FA                           NA
#>   profile_background_image_url_https profile_background_tile
#> 1                                 NA                   FALSE
#>                                                               profile_image_url
#> 1 http://abs.twimg.com/sticky/default_profile_images/default_profile_normal.png
#>                                                          profile_image_url_https
#> 1 https://abs.twimg.com/sticky/default_profile_images/default_profile_normal.png
#>   profile_link_color profile_sidebar_border_color profile_sidebar_fill_color
#> 1             1DA1F2                       C0DEED                     DDEEF6
#>   profile_text_color profile_use_background_image has_extended_profile
#> 1             333333                         TRUE                 TRUE
#>   default_profile default_profile_image following follow_request_sent
#> 1            TRUE                  TRUE     FALSE               FALSE
#>   notifications translator_type withheld_in_countries
#> 1         FALSE            none                  NULL
lookup_users("ConvertedBook")
#> Error in `$<-.data.frame`(`*tmp*`, "user_id", value = "1407791753470496769"): replacement has 1 row, data has 0
```

<sup>Created on 2021-07-14 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

This was introduced in febe1c495c67737f63571f39e1fed028e52bfb40. PR with fix coming shortly.

**Comment by llrs** [comment]
This has already been reported on #574 and is fixed (among other things) by #572.

**Comment by alexpghayes** [comment]
Oh wonderful! Sorry for the duplicate issue!
